boards/shields/amg88xx/doc/index.rst

```diff
@@ -24,7 +24,7 @@ For sensor evaluation and rapid prototyping multiple shields are available:
 
 - `Panasonic Grid-EYE Evaluation Shield`_
 
- .. figure:: AMG88XX_panasonic_grid_eye_evaluation_shield.png
+ .. figure:: amg88xx_grid_eye_eval_shield.png
```

Comment:
It does not seem to be related to "Adopt shield.yml for all the shields in-tree."; perhaps it should be moved to a separate commit.

Reply:
done, thanks.